docs/Automation.md

```diff
@@ -1,183 +1,196 @@
-provided in automation mode, and the code is intended to be compatible with
-Windows 7 or newer.
+To provide automated accessibility testing for Windows applications, we have created the `Axe.Windows.Automation` .NET assembly which exposes a subset of core
+AxeWindows functionality to automation systems.
 
-### General Characteristics
+### How To Run An Automated Scan
 
-#### Fully Synchronous
-Since these commands are all stateful, they are intentionally synchronous within
-a process. If you attempt to call into the commands concurrently, the first one
-to obtain the lock will execute, then another, then another. This is by design
-and is not expected to change at any future time. If you have a scenario that
-truly requires the command to execute in parallel, then you will need to create
-a solution where you can make those calls from separate processes.
+1. Create a `Config` object using `Config.Builder`.
 
-#### Specifying parameters
-Parameters are specified as a string-to-string dictionary. A hierarchical
-parameter structure is supported, as follows:
--   Tier 3 parameters can be read from a JSON-serialized disk file, which is
-    specified when the automation session begins. Each of these parameters
-    will be used unless they are overridden by a corresponding Tier 2 or Tier 1
-    parameter.
--   Tier 2 parameters can be specified via a .NET `Dictionary<string,string>`
-    parameter that is passed to the Start command. Each of these parameters will
-    be used for the lifetime of the automation session unless they are
-    overridden by a Tier 1 parameter.
--   Tier 1 parameters can be passed to some commands. Tier 1 parameters are
-    never overridden.
-
-#### Return Objects
-Each command will return a .NET object to summarize the result of the operation.
-These objects are detailed under each command:
-
-### Command Details
-#### Start Command
-The Start command initializes the AxeWindows engine for automation.
-It is required for the session to begin, and each session requires a
-corresponding call to the Stop command. A given test process is allowed only one
-active session at a time, and subsequent calls to Start will fail.
-
-##### Inputs
-
-###### .NET
-
-The **StartCommand.Execute** method accepts the following parameters:
+        // Create config to specifically target a process
+        var myConfigBuilder = Config.Builder.ForProcessId(1234);
 
-**Name** | **Type** | **Description**
----|---|---
-primaryConfig | `Dictionary<string, string>` | The set of Tier 2 parameters to apply during this session. See [Defined Constants](#defined-constants) for specific parameters.
-configFile | `string` | The full path to a file that contains JSON-serialized Tier 3 parameters. Can be `string.Empty`. Must point to a valid JSON-serialized file if non-empty.
+        // Optional: configure to create an A11yTest file
+        myConfigBuilder.WithOutputFileFormat(OutputFileFormat.A11yTest);
```

Comment:
The docs are accurate, although I was surprised this code works without chaining the calls or saving the return value - I incorrectly assumed the `With` methods return a new builder. It might just be me :)